src/components/atoms/Plugin/IFrame/hooks.ts

```diff
@@ -109,15 +112,15 @@ export default function useHook({
             const width = ${
               width
                 ? typeof width === "number"
-                  ? width
-                  : width + "px"
+                  ? `"${width}px"`
+                  : `"${width}"`
```

Comment:
I don't think you need the "" here or with the height. You are using string literals which wil take the javascript variable and use it in the string.
```suggestion
                  ? `${width}px`
                  : `${width}`
```

Reply:
As we talked, this part does not needs updated.